ContactMechanics/PipelineFunction.py

```diff
@@ -0,0 +1,327 @@
+#
```

Comment:
I don't review this file because I expect it to be mainly copied from Topobank

Reply:
It's not just copied, some things have been rearranged

Reply:
ok I have a quick look right now

Reply:
Okay, thanks